atd-vze/src/Components/GridTableSearch.js

```diff
@@ -104,7 +104,7 @@ const GridTableSearch = ({
               toggle={toggleDropdown}
             >
               <DropdownToggle caret color="secondary">
-                {fieldToSearch === "" ? "Field" : getFieldName(fieldToSearch)}
+                {fieldToSearch === "" ? "Select field..." : getFieldName(fieldToSearch)}
```

Comment:
definitely easier to tell what is going on here because i forgot that this message does show on some tables like Fatalities before a config is saved. 👍